contracts/multisig/src/signing.rs

```diff
@@ -25,19 +24,19 @@ impl SigningSession {
             id: session_id,
             key_id,
             msg,
+            state: MultisigState::Pending,
         }
     }
 }
 
 fn validate_session_signature(
     session: &SigningSession,
-    signatures: &HashMap<String, Signature>,
     key: &Key,
     signer: String,
     signature: &Signature,
 ) -> Result<(), ContractError> {
     // TODO: revisit again once expiration and/or rewards are introduced
```

Comment:
both are introduced now, should this comment change?

Reply:
I'm removing this on the next PR when I'm taking the grace period into account in this `if`